components/interface/VFBCircuitBrowser/VFBCircuitBrowser.js

```diff
@@ -428,6 +438,7 @@ class VFBCircuitBrowser extends Component {
               zoomIn={self.zoomIn}
               zoomOut={self.zoomOut}
               circuitQuerySelected={this.circuitQuerySelected}
+              datasource="SOLR"
```

Comment:
@ddelpiano I added this to this branch to resolve the conflicts between this branch and feature/660.